R/derive_param_clinbenefit.R

```diff
@@ -92,6 +100,12 @@
 #' The (first) argument of the function must expect a character vector and the
 #' function must return a numeric vector.
 #'
+
+#' *Default:* `yn_to_numeric` (see `admiral::yn_to_numeric()` for details)
```

Comment:
This should probably be removed, given the decision to remove all defaults in headers

Reply:
Agreed, will remove